examples/models/llama2/source_transformation/sdpa.py

```diff
@@ -107,6 +108,47 @@ def forward(
         return y.transpose(1, 2).contiguous().view(bsz, seqlen, self.dim)
 
 
+class SDPAQNN(torch.nn.Module):
```

Comment:
We can rename it to something else, it's another sdpa replacement, not necessarily QNN specific

Reply:
Sounds great:)